h2o-py/tests/testdir_misc/pyunit_drop.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+
+
+
+
+def pyunit_drop():
+
+    #Import data and collect number of columns and number of rows
+    pros = h2o.import_file(pyunit_utils.locate("smalldata/prostate/prostate.csv"))
+    nc = pros.ncol
+    nr = pros.nrow
+    print(pros.names)
+
+    #There are two ways to drop a single column: Pass in an int index or a string column,i.e., 1 or "C1"
+    dropped_col_int = pros.drop(0)
+    dropped_col_string = pros.drop("ID")
+
+    #There are two ways to drop a set of columns: Pass in an int array or a string array, i.e., [1,2] or ["C1", "C2"]
+    dropped_col_int_array = pros.drop([0,1])
+    dropped_col_string_array = pros.drop(["ID","CAPSULE"])
+
+    #Drop a first few rows from the frame (0 based)
+    dropped_row_array_0 = pros.drop([0],axis=0)
+    dropped_row_array_1 = pros.drop([0,1],axis=0)
+    dropped_row_array_2 = pros.drop([0,1,2],axis=0)
+
+    #Drop last few rows from the frame (0 based)
+    dropped_row_array_380 = pros.drop([379],axis=0)
+    dropped_row_array_378 = pros.drop([378,379],axis=0)
+    dropped_row_array_377 = pros.drop([377, 378, 379],axis=0)
+
+    #Print dimensions of each frame
+    print(pros.dim)
+    print(dropped_col_int.dim)
+    print(dropped_col_string.dim)
+    print(dropped_col_int_array.dim)
+    print(dropped_col_string_array.dim)
+    print(dropped_row_array_0.dim)
+    print(dropped_row_array_1.dim)
+    print(dropped_row_array_2.dim)
+    print(dropped_row_array_380.dim)
+    print(dropped_row_array_378.dim)
+    print(dropped_row_array_377.dim)
+
+    #Print each frame
+    print(pros)
+    print(dropped_col_int)
+    print(dropped_col_string)
+    print(dropped_col_int_array)
+    print(dropped_col_string_array)
+    print(dropped_row_array_0)
+    print(dropped_row_array_1)
+    print(dropped_row_array_2)
+    print(dropped_row_array_380)
+    print(dropped_row_array_378)
+    print(dropped_row_array_377)
```

Comment:
Okay, that sounds good. I have checks similar to this. Towards the bottom of this test.